.github/workflows/nightly.yml

```diff
@@ -1,21 +1,25 @@
-name: Integration Tests
+name: Nightly Tests
 
 on:
-  pull_request: { }
   push:
-    branches: [ main, develop ]
+    branches: [ release/* ]
```

Comment:
does it mean that it will not run on develop branch, but only on release branches?

Reply:
I _believe_ crons are set to the latest commit on the default branch, this would just be for pushes

Reply:
All pushes to release branches; cron on "the default branch"

Reply:
So the only extra we could have could be on push to the develop branch?